base_cancel_confirm/model/base_cancel_confirm.py

```diff
@@ -25,6 +27,19 @@ class BaseCancelConfirm(models.AbstractModel):
         help="An optional cancel reason",
     )
 
+    def _cancel_confirm_disabled(self):
+        key = "%s.cancel_confirm_disable" % self._name
```

Comment:
I think `_enable` makes more sense, but it's likely a matter of taste :)

Reply:
We use disable for backward compatibility too, there are extended modules what should be enable by default.